HARK/ConsumptionSaving/tests/test_ConsAggShockModel.py

```diff
@@ -150,10 +152,12 @@ def test_methods(self):
         )
 
         self.economy.make_Mrkv_history()  # Make a simulated history of aggregate shocks
-        self.assertAlmostEqual(
-            np.mean(self.economy.MrkvNow_hist),
-            0.4818181818181818
-        )
+
```

Comment:
In conversations with Chris and others, this types of tests was controversial.

Notice is not testing for a single simulation value, but for the mean of a long history of simulations. Chris liked these tests because they are informative (if the mean of 10k simulations is 30% off of where we expect it to be, there is something wrong). You (Seb) disagreed because technically there is a greater than zero probability of that event occurring even if all the code is fine.

I see both points. 

I'd vote for keeping them (because they are informative) but making them have a very loose tolerance. Say, 10% relative tolerance for the mean of a 10k obs simulation

Reply:
However, happy to revise my position if this is heretical from a software engineering/comp-sci point of view